packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-sign-in-pwd-phone/java/integrationsteps.md

```diff
@@ -1,3 +1,100 @@
+```
+
+If the password is validated, the `IDXAuthenticationWrapper.authenticate()` method returns an [`AuthenticationResponse`](https://github.com/okta/okta-idx-java/blob/master/api/src/main/java/com/okta/idx/sdk/api/response/AuthenticationResponse.java) object with the following properties:
+
+* `AuthenticationStatus=AWAITING_AUTHENTICATOR_ENROLLMENT_SELECTION` &mdash; This status indicates that there are required authenticators that need to be enrolled.
+
+* `Authenticators` &mdash; List of [authenticators](https://github.com/okta/okta-idx-java/blob/master/api/src/main/java/com/okta/idx/sdk/api/client/Authenticator.java) to be enrolled (in this case, there is only the phone authenticator). <br>
+    Authenticators are the factor credentials that are owned or controlled by the user. These are verified during authentication.
+
+> **Note:** If the user already has the phone authenticator enrolled, then `AuthenticationStatus=AWAITING_AUTHENTICATOR_SELECTION` is returned (instead of `AuthenticationStatus=AWAITING_AUTHENTICATOR_ENROLLMENT_SELECTION`) and the user does not have to enroll the phone authenticator with a phone number, bypassing steps [3](#step-3-user-selects-phone-authenticator) and [4](#step-4-user-enters-phone-number).
+
+After receiving the `AWAITING_AUTHENTICATOR_ENROLLMENT_SELECTION` status and the list of authenticators to be enrolled, provide the user with a form to select the authenticator to enroll. In the following example, phone is the only authenticator:
+
+<div class="common-image-format">
+
+![Displays the enroll phone authenticator selection page for Java SDK](/img/oie-embedded-sdk/oie-embedded-sdk-use-case-sign-in-pwd-phone-enroll-phone-java.png)
+
+</div>
+
+> **Tip:** Build a generic authenticator selection form to handle single or multiple authenticators returned from the SDK.
+
+### Step 3: User selects phone authenticator
+
+In this use case, the user selects **Phone** as the authenticator to enroll. Pass this selected authenticator to the `IDXAuthenticationWrapper.selectAuthenticator()` method.
+
+```java
+authenticationResponse = idxAuthenticationWrapper.selectAuthenticator(proceedContext, authenticator);
+```
+
+The response from this request is an `AuthenticationResponse` object with `AuthenticationStatus=AWAITING_AUTHENTICATOR_ENROLLMENT_DATA`. This status indicates that the user needs to provide additional authenticator information. In the case of the phone authenticator, the user needs to specify a phone number.
+
+> **Note:** Only SMS is currently supported for the phone authenticator method factor. In the future, if your org supports the voice factor, you will need to add a voice or an SMS factor selection form for your app. Both the phone authenticator and the phone method factor needs to be passed to the [`IDXAuthenticationWrapper.submitPhoneAuthenticator()`](https://github.com/okta/okta-idx-java/blob/master/api/src/main/java/com/okta/idx/sdk/api/client/IDXAuthenticationWrapper.java#L398) method.
```

Comment:
Voice is supported by the SDK, but takes a feature flag to enable. Having voice enabled changes some of the sdk calls you need to make too. I think I've posted something similar to this before, but it has to deal with `submitPhoneAuthenticator`

Reply:
I will add an alternative step 4 in the **Summary of Steps** diagram section, and **Integration Steps** section. This will show the flow and method call if voice FF is enabled.